cartesi-rollups/api/backend/vouchers.md

```diff
@@ -27,10 +28,12 @@ Epoch configuration is crucial when working with vouchers.
 
 Vouchers allow dApps in the execution layer to interact with contracts on the base layer through message calls. A voucher can only be executed once the corresponding epoch is closed. By default, Cartesi nodes close one epoch a day.
```

Comment:
Node on testnet and mainnet will probably not use 1 day, and probably will use 1 week.
So this statement can be misleading.